### DISCUSSION TITLE
是否能够隐藏右上角的图标

### DISCUSSION THREAD

**Discussion by tismega** [spencerwooo/onedrive-vercel-index#167] [Q&A] [answered]
因为用不到右上角的跳转链接，不知道有没有方法隐藏那三个图标

**Comment by spencerwooo** [comment, marked as answer]
这里：https://github.com/spencerwooo/onedrive-vercel-index/blob/main/components/Navbar.tsx#L56-L77

**Reply by tismega** [reply]
非常感谢！
